var/spack/repos/builtin/packages/py-astroid/package.py

```diff
@@ -45,12 +47,14 @@ class PyAstroid(PythonPackage):
     depends_on('py-six@1.12:1', when='@2.3.3:2.7.2', type=('build', 'run'))
     depends_on('py-wrapt', when='@:2.2', type=('build', 'run'))
     depends_on('py-wrapt@1.11:1.12', when='@2.3.3:2.8.2', type=('build', 'run'))
-    depends_on('py-wrapt@1.11:1.13', when='@2.8.3:', type=('build', 'run'))
+    depends_on('py-wrapt@1.11:1.13', when='@2.8.3:2.10', type=('build', 'run'))
+    depends_on('py-wrapt@1.11:1', when='@2.11:', type=('build', 'run'))
     depends_on('py-enum34@1.1.3:', when='^python@:3.3', type=('build', 'run'))
     depends_on('py-singledispatch', when='^python@:3.3', type=('build', 'run'))
     depends_on('py-backports-functools-lru-cache', when='^python@:3.2', type=('build', 'run'))
     depends_on('py-typed-ast@1.3.0:', when='@2.2.5:2.3.0 ^python@3.7.0:3.7')
-    depends_on('py-typed-ast@1.4.0:1.4', when='@2.3.1: ^python@:3.7', type=('build', 'run'))
+    depends_on('py-typed-ast@1.4.0:1.4', when='@2.3.1:2.8.4 ^python@:3.7', type=('build', 'run'))
+    depends_on('py-typed-ast@1.4.0:1.9', when='@2.8.5: ^python@:3.7', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('py-typed-ast@1.4.0:1', when='@2.8.5: ^python@:3.7', type=('build', 'run'))
```
In case a 1.10 release comes out